src/containers/TopbarContainer/Topbar/TopbarDesktop/TopbarDesktop.js

```diff
@@ -55,28 +51,30 @@ const InboxLink = ({ notificationCount, currentUserHasListings }) => {
   );
 };
 
-const ProfileMenu = ({ currentPage, currentUser, onLogout }) => {
+const ProfileMenu = ({ currentPage, currentUser, onLogout, showManageListingsLink }) => {
   const currentPageClass = page => {
     const isAccountSettingsPage =
       page === 'AccountSettingsPage' && ACCOUNT_SETTINGS_PAGES.includes(currentPage);
     return currentPage === page || isAccountSettingsPage ? css.currentPage : null;
   };
 
+  const manageListingsLinkMaybe = showManageListingsLink ? (
+    <NamedLink
+      className={classNames(css.menuLink, currentPageClass('ManageListingsPage'))}
+      name="ManageListingsPage"
+    >
+      <span className={css.menuItemBorder} />
+      <FormattedMessage id="TopbarDesktop.yourListingsLink" />
+    </NamedLink>
+  ) : null;
+
   return (
     <Menu>
       <MenuLabel className={css.profileMenuLabel} isOpenClassName={css.profileMenuIsOpen}>
         <Avatar className={css.avatar} user={currentUser} disableProfileLink />
       </MenuLabel>
       <MenuContent className={css.profileMenuContent}>
-        <MenuItem key="ManageListingsPage">
-          <NamedLink
-            className={classNames(css.menuLink, currentPageClass('ManageListingsPage'))}
-            name="ManageListingsPage"
-          >
-            <span className={css.menuItemBorder} />
-            <FormattedMessage id="TopbarDesktop.yourListingsLink" />
-          </NamedLink>
-        </MenuItem>
+        <MenuItem key="ManageListingsPage">{manageListingsLinkMaybe}</MenuItem>
```

Comment:
It sounds like this creates empty MenuItesm (`<li>` elements) to the menu:
`<MenuItem key="ManageListingsPage">{null}</MenuItem>`

Does this cause errors if you create the complete MenuItem conditionally?
```jsx
{showManageListingsLink ? (
  <MenuItem key="ManageListingsPage">
    <NamedLink
        className={classNames(css.menuLink, currentPageClass('ManageListingsPage'))}
        name="ManageListingsPage"
      >
        <span className={css.menuItemBorder} />
        <FormattedMessage id="TopbarDesktop.yourListingsLink" />
      </NamedLink>
  </MenuItem>
) : null}
```

I.e. I wonder if the null ends up here: 
https://github.com/sharetribe/web-template-private/blob/main/src/components/MenuContent/MenuContent.js#L53

Reply:
Yes, creating the complete MenuItem conditionally breaks things because of that `child.type` check.

Reply:
Updated the child checks in MenuContent to allow null and undefined children